MuxExoPlayer/src/main/java/com/mux/stats/sdk/muxstats/MuxNetworkRequests.java

```diff
@@ -225,14 +322,29 @@ private boolean executeHttp(URL url, String method, Hashtable<String, String> he
     }
   }
 
-  private String getAuthority(String propertykey) {
+  /**
+   * Calculate the backend URL based on the user environment key variable.
+   *
+   * @param propertykey environment key variable.
+   * @param domain domain to send beacons to (concatenated with propertykey).
+   * @return backend url.
+   */
+  private String getAuthority(String propertykey, String domain) {
     if (Pattern.matches("^[a-z0-9]+$", propertykey)) {
-      return propertykey + ".litix.io";
+      if (domain.startsWith(".")) {
```

Comment:
This is what I was saying about ensuring that from Core we should handle the prepended `.` or not, so this code in here doesn't have to handle it. It's okay, but ideally we wouldn't make the consuming application deal with potential inconsistencies coming from our Core SDK.

Reply:
@jsanford8 the change to MuxCore to check for `.` is already in v7.0.0. I'll remove the check here.

Reply:
Ahh perfect, good call. Sorry, I somehow missed that last night!